docs/getting-started.md

```diff
@@ -18,7 +18,10 @@ In this workshop, we’ll build an app with quite a few features, including:
 
 Here’s a map of the services we’ll use and how they’ll all connect.
 
-< TODO >
+<img src="../images/architecture.png"
```

Comment:
Did a quick diagram to knock out your TODO - it is using draw.io and the diagram source itself is embedded in the image, so it should be easy to update/edit/change in the future.